index.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
<!--
  0 failure: 
  0 warning: 
  1 messages
  
  DangerID: danger-id-default;
  File: index.js;
  Line: 1;
-->  


- :book: 'use strict' is unnecessary inside of modules. (strict)